packages/eslint-config-react-native-community/index.js

```diff
@@ -53,7 +53,10 @@ module.exports = {
       rules: {
         '@typescript-eslint/no-unused-vars': [
           'error',
-          {argsIgnorePattern: '^_'},
+          {
+            argsIgnorePattern: '^_',
+            destructuredArrayIgnorePattern: '^_',
```

Comment:
Looks like this is only available after [@typescript-eslint/eslint-plugin](https://togithub.com/typescript-eslint/typescript-eslint) 5.17.0, while the current version required is [`^5.15.0`](https://github.com/facebook/react-native/blob/3e5503eaa722fa317212caf370415225e6da4257/packages/eslint-config-react-native-community/package.json#L15). Could you also update that?

Reply:
Done